File: MVC_DiceGame/MVC_DiceGame_csharp/DiceGame/view/Console.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DiceGame.view
{
	class Console
	{
        private char keyToExit = 'q';

		public void DisplayInstructions()
		{
			System.Console.Clear();
			String message = $"Welcome to the cool Dice Game. Press any Key to play, or { keyToExit } to Quit";
			System.Console.WriteLine(message);
		}

		public bool WantsToPlay()
		{
			return System.Console.ReadKey().KeyChar != keyToExit;
		}

		public char GetInput()
		{
			return System.Console.ReadKey().KeyChar;
		}

		public void DisplayResult(bool a_isWinner, int a_dice1Value, int a_dice2Value)
		{
			if (a_isWinner)
			{
				System.Console.WriteLine($"You are a winner: { a_dice1Value } { a_dice2Value }");
			}
			else
			{
				System.Console.WriteLine($"Sorry you lost: { a_dice1Value } { a_dice2Value }");
			}
		}
	}
}

File: MVC_DiceGame/MVC_DiceGame_ruby/view/Console.rb
module View
  class Console

    QUIT_ON_INPUT  = "q"

    def display_instructions
      system "clear"
      puts "Welcome to the cool Dice Game. Press any Key to play, or #{QUIT_ON_INPUT} to Quit"
    end

    def wants_to_quit?
      gets == "#{QUIT_ON_INPUT}\n"
    end

    def display_result(player_won, dice_one_value, dice_two_value)
        if player_won
          puts "You are a winner: #{dice_one_value} #{dice_two_value}"
        else
          puts "Sorry you lost: #{dice_one_value} #{dice_two_value}"
        end
    end

  end
end
